src/Parser/RetributionPaladin/Modules/Items/LiadrinsFuryUnleashed.js

```diff
@@ -22,8 +21,10 @@ class LiadrinsFuryUnleashed extends Module {
 		const hpGained = this.holyPowerTracker.generatedAndWasted[SPELLS.LIADRINS_FURY_UNLEASHED_BUFF.id].generated;
 		return {
 			item: ITEMS.LIADRINS_FURY_UNLEASHED,
-			result: (
-				`${formatNumber(hpGained)} Holy Power gained from Liadrin's Fury Unleased.`
+			result: (<dfn data-tip={`
+				Total Holy Power Gained: ${formatNumber(hpGained)}`}>
+				{formatNumber(hpGained / this.owner.fightDuration * 60000)} Holy Power gained per minute from Liadrin's Fury Unleased.
```

Comment:
More of a format thing. The portion `from Liadrin's Fury Unleased.` is potentially redundant because this is about the specific item.